examples/window-app/efr32/src/WindowAppImpl.cpp

```diff
@@ -336,12 +370,16 @@ void WindowAppImpl::DispatchEvent(const WindowApp::Event & event)
         break;
     case EventId::CoverChange:
         mIconTimer.Start();
+        #ifdef DISPLAY_ENABLED
```

Comment:
Actually there is another PR doing this display enable handling
please wait for it to be merged first and rebase